### PR TITLE
Color DevTools footer to indicate app connection state

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Part of https://github.com/flutter/devtools/issues/5990, and a follow up change to https://github.com/flutter/devtools/pull/6010. See  flutter.dev/go/static-tooling-in-devtools for original proposal.

When an app is connected, the footer will be colored with the DevTools primary color. When no app is connected, the footer will be the same color as the DevTools background.

static-tooling-with-footer

CC @nancyhu2023 @InMatrix

### CODE REVIEW

**Comment by polina-c** [comment]
Cool!

Minor thing: it looks as if App Size disappears, but it is still here on the very right.
Are you sure we want to reorder tabs?
Users remember tab order and reordering will create confusion.

File: packages/devtools_app/lib/src/framework/report_feedback_button.dart [hunk 1]
```diff
@@ -14,7 +14,9 @@ import '../shared/globals.dart';
 import '../shared/theme.dart';
 
 class ReportFeedbackButton extends StatelessWidget {
```
Comment (polina-c) [inline packages/devtools_app/lib/src/framework/report_feedback_button.dart:16]: Can you add doc comment for this widget?

**Comment by polina-c** [comment]
Would it make sense to check color before and after connection in devtools_app/integration_test/test/live_connection/app_test.dart?

**Comment by kenzieschmoll** [comment]
> Would it make sense to check color before and after connection in devtools_app/integration_test/test/live_connection/app_test.dart?

Done.

> Minor thing: it looks as if App Size disappears, but it is still here on the very right.
Are you sure we want to reorder tabs?
Users remember tab order and reordering will create confusion.

This discussion should likely happen on https://github.com/flutter/devtools/pull/6010, since that PR is where this change is implemented. Moving the discussion there.

**Review by elliette — APPROVED** [review]

**Pull request merged**